### ISSUE TITLE
Consider introducing Issue Templates

### ISSUE THREAD

**Issue by csarven** [solid/specification#546]
It may help to introduce some issue templates.

Let's process https://github.com/solid/specification/blob/main/CONTRIBUTING.md (e.g., Work Item Proposals) and other material elsewhere to see what would be useful to template.

* New Work Item Proposal
* Use Case Proposal - This may need to be in https://github.com/solid/user-stories or Work Item repo specific?
* Work Item Change Proposal - identify intended/anticipated change using [W3C correction classes](https://www.w3.org/2023/Process-20230612/#correction-classes) (issue should be for "significant" classes 3-5).
* Dedicated Meeting Proposal (topic, time, location, focus..)
* ...

**Comment by woutermont** [comment]
Some other possible templates:

- implementation questions: what is the context/setup, what is the question, what has already been done to find an answer (where was the answer expected) ...

- implementation feedback (which not yet resulted in concrete work item change request): what was the context/setup, did everything conform, what went wrong or could be better ...

- demo (to document): what was the context/setup, what does it demo, where can we find the demo ...

---

Re issues that should be raised somewhere else (e.g. use cases): can the template picker display some references next to the available templates?

**Comment by elf-pavlik** [comment]
Yes, it can have a link, see for example: https://github.com/solid/authorization-panel/issues/new/choose

**Comment by woutermont** [comment]
> Yes, it can have a link, see for example: https://github.com/solid/authorization-panel/issues/new/choose

This is just perfect design; makes me happy 😊
